java-checks/src/main/java/org/sonar/java/checks/MutableMembersUsageCheck.java

```diff
@@ -52,7 +52,7 @@ public class MutableMembersUsageCheck extends BaseTreeVisitor implements JavaFil
   private static final List<String> MUTABLE_TYPES = Arrays.asList(
     "java.util.Collection",
     "java.util.Date",
-    "java.util.Hashtable");
+    "java.util.Map");
```

Comment:
Becuase we now raise on `Map`, we should add `com.google.common.collect.ImmutableMap` to the IMMUTABLE_TYPES list.

Reply:
Yes, I was sure it's already included 🤔